databrowser/src/domain/cellComponents/components/cells/dateCellOrdinal/DateCellOrdinal.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <span v-if="date != null">{{ formattedDate }}</span>
+</template>
+
+<script setup lang="ts">
+import { computed, defineProps, toRefs, withDefaults } from 'vue';
+
+const props = withDefaults(
+  defineProps<{
+    date?: string;
+    format?: string;
+  }>(),
+  {
+    date: undefined,
+    format: undefined,
+  }
+);
+
+const { date, format } = toRefs(props);
+
+const nth = function (d) {
+  if (d > 3 && d < 21) return 'th';
+  switch (d % 10) {
+    case 1:
+      return 'st';
+    case 2:
+      return 'nd';
+    case 3:
+      return 'rd';
+    default:
+      return 'th';
+  }
+};
+
+console.log(date.value);
+
+const formattedDate = computed(() => {
+  if (format == null) {
+    return date;
+  }
+  if (date != null) {
+    const fullDate = new Date(Date.parse(date.value));
+    const day = fullDate.getDate();
+    const month = [
+      'January',
+      'February',
+      'March',
+      'April',
+      'May',
+      'June',
+      'July',
+      'August',
+      'September',
+      'October',
+      'November',
+      'December',
+    ][fullDate.getMonth()];
+    const year = fullDate.getFullYear();
+
+    return day + nth(day) + ' ' + month + ' ' + year;
+  }
+  return '';
+});
```

Comment:
Check if you could use date-fns for date formatting (https://date-fns.org/v2.29.2/docs/format)